UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -237,11 +237,9 @@ public Layer BGColorLayer
     {
         get
         {
-            return _layers?.Where(l => l.LayerType is LayerType.Background
-                                       && l.BackgroundData.Sprite is null
-                                       && l.BackgroundData.Color != 0)
-                           .OrderBy(l => l.LayerDepth)
-                           .FirstOrDefault();
+            return (_layers?.Where(l => l.LayerType is LayerType.Background
+                                        && l.BackgroundData.Sprite is null
+                                        && l.BackgroundData.Color != 0)).MinBy(l => l.LayerDepth);
```

Comment:
Nice suggestion, I just forgot about this method.
But what if:
```suggestion
            return (_layers?.Where(l => l.LayerType is LayerType.Background
                                        && l.BackgroundData.Sprite is null
                                        && l.BackgroundData.Color != 0))
                            .MinBy(l => l.LayerDepth);
```

Reply:
sure